src/status_im/browser/core.cljs

```diff
@@ -179,7 +179,7 @@
                                assoc
                                :url (str gateway path)
                                :resolving? false)
-                       (assoc-in [:browser/options :resolved-ens host] gateway))})))
+                       (assoc-in [:browser/options :resolved-ens host] (string/replace gateway #"https://" "")))})))
```

Comment:
nah if you are going to remove the https prefix just don't put it there in the first place

`(str "https://" gateway path)`